src/rprblender/export/mesh.py

```diff
@@ -112,6 +113,38 @@ def init_from_mesh(mesh: bpy.types.Mesh, calc_area=False, obj=None):
 
         if calc_area:
             data.area = sum(tri.area for tri in mesh.loop_triangles)
+           
+        # Not possible to store strings/ bools in primvars. If the type was bool, it would be possible to just pass 1 or 0, and let it be interpolated, and then round it, but that would require extra code in the shaderattribute node
+        # Also cannot currently handle anything besides vertex attributes (point/corner), without triangulating the mesh first, which is over my head.
+        # Ints and Int8s are currently excluded as well, and byte color is stored as float anyway(?)
+        # Ignoring givens: uvmap and position
+        # Returning list of lists, [AttName, AttSize, AttData, AttInterop] Interop stored for future implementation Face/Edge types (im assuming which are far less common)
+        validAtts =  [e for e in mesh.attributes if e.data_type in ['FLOAT', 'FLOAT_VECTOR', 'FLOAT_COLOR', 'BYTE_COLOR', 'FLOAT2'] 
```

Comment:
again use snake case valid_attrs for variable names

Reply:
This was a slip by me, thanks. I normally write in camelcase with underscores for prefixing the "class" the function would belong to, lol.